tests/00-fmt.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env sh
 
-set -ex
+set -x
```

Comment:
What's failing on the diff that make you move the `set -e` to line 11?

Reply:
the bindata file does not exist when this is ran in the Ginkgo suite. When we were running this in the current build setup, it does exist. Since we are inspecting the output of the command, we don't need `set -e` for it anyway.